### DISCUSSION TITLE
How to exclude Addy from ProtonMail spam filter?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#660] [Q&A] [answered]
I just had a failed delivery due to https://github.com/anonaddy/anonaddy/discussions/194#discussioncomment-1289624. I'd rather like to get this fixed, since I've begun to use Addy's rulesets rather than Proton's for blocking addresses, but still need nominal protection for my ultimate address.

**Comment by willbrowningme** [comment]
I'm not sure exactly what you are referring to to get "fixed".

**Reply by roke-julian-lockhart** [reply]
Fixing on my end the fact that my spam filter is working on AnonAddy. It shouldn't be, as you explained – I'd rather not reduce its reputation.

I imagine a user here has already done this.

**Reply by willbrowningme** [reply, marked as answer]
In Proton you can add an entire domain to your allow list from this page [https://account.proton.me/u/0/mail/filters](https://account.proton.me/u/0/mail/filters)

> Allow: To ensure critical messages don't end up in spam and getting missed
